### ISSUE TITLE
bl download requires a login while I can download from brainlife.io by just agreeing to terms

### ISSUE THREAD

**Issue by yarikoptic** [brain-life/cli#41]
I was trying to get
https://brainlife.io/pub/5a0f0fad2c214c9ba8624376#5a065cc75ab38300be518f51

```shell
(git-annex)hopa:~/proj[master]brainlife
$> bl dataset download --id 5a065cc75ab38300be518f51
(node:22532) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Couldn't find your access token. Please try logging in by running 'bl login'
(node:22532) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

$> bl login
username:  ...
```
while I can click "download" on brainlife web ui and then agree to HCP terms and proceed

**Comment by soichih** [comment]
Yes, at the moment, our command line interface requires our users to authenticate - even though they might be downloading publically downloadable datasets.

It's really simple to sign up to Brainlife, so at the moment this is somewhat considered to be a "feature" that allows us to do better auditing (to know who is doing the download).. but I understand that it's inconsistent.

**Comment by yarikoptic** [comment]
could you remind  me which  email i have used for `yarikoptic` login so I could recover my password? ;-)

**Comment by soichih** [comment]
debian@onerussian.com

It looks like you also used your github account to authenticate - you should be able to? You can reset your password under settings / account page.

**Comment by yarikoptic** [comment]
Great, thanks - I will try later on!
